### ISSUE TITLE
What version of scala/spark do I need?

### ISSUE THREAD

**Issue by jayavanth** [ZuInnoTe/spark-hadoopcryptoledger-ds#19]
I couldn't find the jars in spark-1.5.0 so I downloaded spark-2.12 and changed the sbt file
```
-libraryDependencies += "org.apache.spark" %% "spark-core" % "1.5.0" % "provided"
+libraryDependencies += "org.apache.spark" %% "spark-core" % "2.1.2" % "provided"
```

I get this error when I copy the `2.12` jars into `project/target/scala-2.12`

```
sbt.librarymanagement.ResolveException: unresolved dependency: org.apache.spark#spark-core_2.12;2.1.2: not found
```

**Comment by jornfranke** [comment]
Spark does not support scala 2.12. Only scala 2.11 / scala 2.10

**Comment by jornfranke** [comment]
You need to define the Spark version in the build.sbt
Spark 2.4 will support Scala 2.12
We will support Scala 2.12 in the next release. Meanwhile use Scala 2.11 (any recommended, because Spark 2.4 was just released and is the first version to support 2.12)
